<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Syringe Pump Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>501</width>
      <height>323</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="6" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Current plunger position</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Syringe volume (mL)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="SyringeVolumeSpinBox">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Syringe internal diameter (mm)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="SyringeInternalDiameterSpinBox">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Calibration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="CalibrationSpinBox">
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="SavedLabel">
           <property name="text">
            <string>Saved:?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Step2ApplyButton">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Manual control</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QPushButton" name="ManualWithdrawButton">
           <property name="text">
            <string>Withdraw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ManualInfuseButton">
           <property name="text">
            <string>Infuse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Per microstep:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="VolumeDisplacementPerMicrostepLabel">
         <property name="text">
          <string>---</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>The volume displacement per step:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="VolumeDisplacementPerStepLabel">
         <property name="text">
          <string>-----</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="EngageButton">
           <property name="text">
            <string>Engage motor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DisengageButton">
           <property name="text">
            <string>Disengage motor</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Displacement volume (ml)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="DisplacementVolumeSpinBox">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>2.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Plunger throw (mm)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="PlungerThrowSpinBox">
           <property name="minimum">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Step1CalculateButton">
         <property name="text">
          <string>Calculate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="DisplacementTargetVolumeSpinBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="RampSpinBox"/>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="SpeedUnitComboBox">
         <item>
          <property name="text">
           <string>ul/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ul/min</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ml/s</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ml/min</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="DisplacementTargetVolumeComboBox">
         <item>
          <property name="text">
           <string>ul</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ml</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>s</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="SpeedSpinBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Pumping rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Ramp time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Displacement target volume</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QDoubleSpinBox" name="CurrentPlungerPositionSpinBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="CurrentPlungerPositionSetButton">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="PauseButton">
           <property name="text">
            <string>Pause/Continue</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="WithdrawTargetButton">
           <property name="text">
            <string>Withdraw to target</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="StopButton">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="InfuseTargetButton">
           <property name="text">
            <string>Infuse to target</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1) Calculate syringe diameter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2) Define syringe</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
